crates/bitwarden/Cargo.toml

```diff
@@ -37,6 +39,7 @@ chrono = { version = ">=0.4.26, <0.5", features = [
 getrandom = { version = ">=0.2.9", features = ["js"] }
 hkdf = ">=0.12.3, <0.13"
 hmac = ">=0.12.1, <0.13"
+instant = ">=0.1.12, <0.2"
```

Comment:
Now that we changed the token renewal instants to timestamps this dependency isn't needed anymore, but we probably need to add the `wasmbind` feature to `chrono`